packages/gatsby-cli/.babelrc

```diff
@@ -1,3 +1,20 @@
 {
-  "presets": [["babel-preset-gatsby-package"]]
+  "presets": [
+    [
+      "@babel/env",
+      {
+        "modules": false,
+        "shippedProposals": true,
+        "targets": { "node": "10.13.0" }
```

Comment:
Any reason this is hard coded? Will we have to worry about updating this continuously?

Reply:
Originally this was copied from already existing `gatsby-recipes` bundling setup: https://github.com/gatsbyjs/gatsby/blob/e829ab9eb44e28961e9557538693fd752b221f9f/packages/gatsby-recipes/.babelrc.json#L3 which also has this hardcoded (I didn't put much thought on this TBH).

We also have 10.13.0 hardcoded in few places that just won't allow to DRY it - like in `package.json` -> `engines.node`: https://github.com/gatsbyjs/gatsby/blob/e829ab9eb44e28961e9557538693fd752b221f9f/packages/gatsby/package.json#L188-L190

Only time we need to update it is when we bump minimal supported node version, which is usually done trough mass search and replace for the most part (at least those few times we did it so far).

Reply:
Sounds good. Just wanted to confirm.